_posts/2024-08-20-release-0_7_0.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title:  "Kroxylicious release 0.7.0"
+date:   2024-08-20 00:00:00 +0000
+author: "Keith Wall"
+author_url: "https://www.github.com/k-wall"
+categories:  [releases, kroxylicious, record-validation]
+---
+
+The Kroxylicious project is very pleased to announce the release of [Kroxylicious 0.7.0](https://github.com/kroxylicious/kroxylicious/releases/tag/v0.7.0). See the [Changelog](https://github.com/kroxylicious/kroxylicious/blob/main/CHANGELOG.md#070) for a list of changes and summary of Deprecations, Changes and Removals.
+
+### Record Validation 
+
+In this release, Kroxylcious dips its toes (or, should that be claws?) into the world of record validation.  You can now ensure that the content your producers send adheres to the schema you expect.   If a producer sends
+a message that doesn't meet the expected schema, the producer will get an error message and the message won't be delivered to the topic. Banish those poison messages from your topics for good!
```

Comment:
```suggestion
a record that doesn't meet the expected schema, the producer will get an error message and the record won't be delivered to the topic. Banish those poison records from your topics for good!
```

I found it a little tricky to read since we talk about message and error message in close proximity.

Reply:
```suggestion
a message that doesn't meet the expected schema, the producer will get an error response and the message won't be delivered to the topic. Banish those poison messages from your topics for good!
```

maybe we should use the Kafka term `record` rather than message?